pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>org.openhab.tools</groupId>
   <artifactId>static-code-analysis</artifactId>
-  <version>0.0.3</version>
+  <version>0.0.4-SNAPSHOT</version>
```

Comment:
You would have to change the sat.version in the OH2 pom.xml to the snapshot version as well then. Or you locally change this line here back for your testing.
In general, I agree with @martinvw that it is cleaner to have a snapshot on the latest master (I just didn't care about it as we do not have any CI plan set up for this).